draft-ietf-oauth-status-list.md

```diff
@@ -607,9 +607,9 @@ The Relying Party SHOULD send the following Accept-Header to indicate the reques
 
 If the Relying Party does not send an Accept Header, the response type is assumed to be known implicit or out-of-band.
 
-## Status List Response
+## Status List Response {#status-types-response}
```

Comment:
Why is the reference called status-**types**-response?

Reply:
copy paste mistake - fixed it